lib/node_modules/@stdlib/math/base/special/sec/README.md

```diff
@@ -0,0 +1,204 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Secant
+
+> Evaluate the [secant][trigonometric-functions] of a number.
+
+<section class="intro">
+
+</section>
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var sec = require( '@stdlib/math/base/special/sec' );
+```
+
+## sec( x )
+
+Evaluates the [secant][trigonometric-functions] of `x` (in radians).
+
+```javascript
+var v = sec( 0.0 );
+// returns 1.0
+
+v = sec( 3.141592653589793/2.0 );
+// returns Infinity
```

Comment:
Is the value actually supposed to equal infinity? Given finite precision, IIRC, `cos(pi/2)` does not equal `0`.